jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFMetadataProcessor.java

```diff
@@ -45,11 +45,10 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
                 jsf = servlet;
             }
         }
-        if(jsf != null) {
-            if(jsf.getMultipartConfig() == null) {
-                //WFLY-2329 File upload doesn't work
-                jsf.setMultipartConfig(new MultipartConfigMetaData());
-            }
+        if (jsf != null
+                && jsf.getMultipartConfig() == null) {
```

Comment:
A nitpick, these kinds of checks can be done in a single line. The idea of splitting them into several lines when you have multiple checks is to facilitate knowing which condition fails when we are debugging. However, checks like x != null && x.something does not require multiple lines, because if x is null, you will get an NPE. What's the point of being able to evaluate first if x != null is you know that the check gets an NPE?